src/config/musl.md

```diff
@@ -0,0 +1,94 @@
+# musl
+
+[musl](https://musl.libc.org/) is a libc implementation which strives to be
+lightweight, fast, simple, and correct.
+
+Void officially supports musl by using it in its codebase for all target
+platforms except i686. Additionally, all the packages in our official
+repositories are available with musl-linked binaries in addition to their glibc
+counterparts.
+
+Currently, there are nonfree and debug subrepositories for musl, but no multilib
+subrepo.
+
+Generally speaking, a musl based system is a good choice for servers, embedded
+solutions and old PCs, but it may not be worth the effort to use on a modern
+desktop computer.
+
+## Possible problems
+
+In some cases, software compatibility may be worse with musl.
+
+Proprietary software rarely supports non-glibc libc implementations, although
+sometimes such implementations may be available via
+[flatpak](https://flatpak.org/).
+
+The `gcompat` package, a compatibility layer to allow running glibc binaries on
+musl systems, can sometimes assist in getting glibc-based software to work.
+However, if that doesn't work, try creating a glibc chroot.
+
+### glibc chroot
+
+Create the directory that will contain the chroot:
+
+```
+$ mkdir ~/void-glibc
+```
+
+Install a base system in it:
+
+```
+# xbps-install -S -R https://alpha.de.repo.voidlinux.org/current -r ~/void-glibc base-voidstrap
+```
+
+Mount the appropriate directories:
+
+```
+# mount --bind ~/void-glibc ~/void-glibc # some programs work incorrectly without this
```

Comment:
what is this even about?

Reply:
i don't know; this was just part of what i ported across from the wiki. Should i just delete this line?

Reply:
i can't think of any real cases where this would matter (I never had issues chrooting without creating a new mountpoint) but hard to say for sure

Reply:
Okay, i've removed it - it can always be added back in later if it turns out to be necessary.